locales/en/messages.json

```diff
@@ -4154,5 +4154,8 @@
     },
     "flashTab": {
         "message": "Update Firmware"
+    },
+    "cliAutoComplete": {
+        "message": "Client-side CLI AutoComplete"
```

Comment:
This is for "normal" users. They don't understand "client-side" or similar. I think is better to put something like "Advanced AutoComplete in CLI" or "Contextual AutoComplete in CLI", that will be better understood.

Reply:
I agree. I was going to ask exactly about this but forgot.
@mikeller what would you propose?

I'll push another commit in a while. Found few bugs that affect if turning on/off this feature while the CLI is opened.

Reply:
I'd also think 'advanced auto-complete' is the right thing from the users' perspective - they don't care much where it is generated.